test/fixture/policy/prohibit_implicit_scope_variable_invalid.vim

```diff
@@ -8,7 +8,7 @@ redir END
 let count = 110
 
 function! ImplicitGlobalFunc(param)
-    " Make fix missing a: easy
+    " Make it easy to fix missing a:
```

Comment:
This file should collect only implicit cases.
So, https://github.com/rhysd/vint/blob/fix-ProhibitImplicitScopeVariable/test/fixture/policy/prohibit_implicit_scope_variable_invalid.vim#L10-L19 should be removed from the file (and it move to `prohibit_implicit_scope_variable_valid.vim` but it was already done, thanks :smiley: ).